model-optimizer/CMakeLists.txt

```diff
@@ -6,26 +6,26 @@ if(NOT ENABLE_PYTHON)
 else()
     add_custom_target(model_optimizer DEPENDS ie_api offline_transformations_api ir_ov_frontend)
     if(ENABLE_TESTS)
-            add_subdirectory(unit_tests/mock_mo_frontend/mock_mo_ov_frontend)
-            add_dependencies(model_optimizer mock_mo_ov_frontend)
+        add_subdirectory(unit_tests/mock_mo_frontend/mock_mo_ov_frontend)
+        add_dependencies(model_optimizer mock_mo_ov_frontend)
 
-            add_subdirectory(unit_tests/mock_mo_frontend/mock_mo_python_api)
-            add_dependencies(model_optimizer mock_mo_python_api)
+        add_subdirectory(unit_tests/mock_mo_frontend/mock_mo_python_api)
+        add_dependencies(model_optimizer mock_mo_python_api)
     endif()
 endif()
 
 # install
-ie_cpack_add_component(model_optimizer)
 
-configure_file(
-    "${CMAKE_CURRENT_SOURCE_DIR}/automation/version.txt.in"
-    "${CMAKE_CURRENT_SOURCE_DIR}/version.txt"
-    @ONLY)
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/automation/version.txt.in"
+               "${CMAKE_CURRENT_SOURCE_DIR}/version.txt" @ONLY)
```

Comment:
This change is not necessary (not a part of described problem).
Previous coding style is preferable to maintain clear git history (1 file entry per line).